mongo.js

```diff
@@ -37,15 +37,13 @@ Cursor.prototype._save = function (object, callback) {
 
         function ondone(err, result) {
             conn.done();
+
             if (err) return callback(toError(err));
-            if (typeof result == 'object') {
-                if (result.ops && result.ops.length > 0) {
-                    result = result.ops[0];
-                }
-                result.id = fromObjectID(result._id);
-                delete result._id;
-                replace(object, result);
+
```

Comment:
```suggestion
if (typeof result == 'object') {
```
Is redundant ?

Reply:
Yes. The returned value is always an object. See the docs I linked in the description.